include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -203,15 +203,15 @@ def BoundedModelCheckingOp : VerifOp<"bmc", [
     match the types yielded by the `init` region).
   }];
 
-  let arguments = (ins I32Attr:$bound, I32Attr:$num_regs);
+  let arguments = (ins I32Attr:$bound, I32Attr:$num_regs, ArrayAttr:$initial_values);
```

Comment:
This makes `num_regs` redundant since it's always `num_regs == initialValues.size()`. Could you remove and add `iint32_t getNumRegs() { return getInitialValues.size(); }` to external declaration of the op?